test/e2e/rte/metrics.go

```diff
@@ -139,7 +142,7 @@ var _ = ginkgo.Describe("[RTE][Monitoring] metrics", func() {
 			}
 			rteContainerName, err := e2ertepod.FindRTEContainerName(rtePod)
 			gomega.Expect(err).ToNot(gomega.HaveOccurred())
-			cmd := []string{"curl", "-L", fmt.Sprintf("http://127.0.0.1:%d/metrics", metricsPort)}
+			cmd := []string{"curl", "-L", fmt.Sprintf("https://%s:%d/metrics", MetricsAddress, metricsPort)}
```

Comment:
here we want to test over http (plain text) but you changed the protocol part to "httpS" (emphasis added), which is wrong in the context of this test

Reply:
Ah, good catch!!